modules/ROOT/pages/custom-rulesets-validate-and-publish.adoc

```diff
@@ -0,0 +1,58 @@
+= Validating and Publishing Custom Rulesets
+
+After you create a custom ruleset:
+
+. <<validate-ruleset,Validate>> the changes using the `governance:ruleset:validate` command.
+. <<generate-ruleset-doc,Generate>> the ruleset documentation. 
+. <<publish-ruleset,Publish>> your custom ruleset as a new asset using the Exchange UI or the `anypoint-cli exchange asset upload` command. 
++
+See xref:custom-rulesets-validate-and-publish.adoc[].
+
+For an example of the sequence of commands from validation to publishing, see <<example-publish-sequence>>.
+
+NOTE: As with custom code and configurations, rulesets are not considered supported MuleSoft products. For help with issues with custom rulesets, post an issue in the https://github.com/aml-org/amf-custom-validator[AMF Custom Validator Github repository^].
```

Comment:
> As with custom code and configurations, rulesets are not considered supported MuleSoft products
Do we want to say this?

Reply:
I think we have to, because it was given to us by Support and Legal. I'll check with Ankita and Venkat.

Reply:
I'm leaving it here for now. If we learn we can remove it later, I'll do a separate PR for it.